CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+## 0.9.5 - 2024-09-11
+### Fixed
+* [[915]](https://github.com/microsoft/vscode-azureresourcegroups/pull/915) Remove bad implementation of `createSubscriptionContext`
```

Comment:
Can you link an issue related to the US Government/China clouds for this? We should mention those issues since those were the most prevalent problems due to the bad impl of `createSubscriptionContext`.

Also, are we planning to release this on Wednesday? We can probably release today or tomorrow, yeah?

Reply:
Yeah we should make the changelog more user friendly. We want to communicate with the users what issue has been fixed, not what solution was applied. For us the engineers I think linking the PR is enough info for us.

Reply:
Yeah sure I'll link the issue as well.

As for release plan I'll reach out to discuss further offline

Reply:
Also discussed offline, but the candidate is still in progress for testing, so we'll have to wait a little bit longer before we figure out he final release date